src/librustc/diagnostics.rs

```diff
@@ -749,6 +749,54 @@ for v in &vs {
 ```
 "##,
 
+E0277: r##"
+You tried to called a trait function on a type which doesn't implement this
```

Comment:
You tried to use a type which doesn't implement some trait in a place which expected that trait.